base/applications/mspaint/main.cpp

```diff
@@ -115,193 +82,171 @@ OFNHookProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
     return 0;
 }
 
-/* entry point */
+BOOL CMainWindow::DoGetOpenFileName(IN OUT LPTSTR pszFile, INT cchMaxFile)
+{
+    static OPENFILENAME ofn = { 0 };
+    static CString strFilter;
+
+    if (ofn.lStructSize == 0)
+    {
+        // The "All Files" item text
+        CString strAllPictureFiles;
+        strAllPictureFiles.LoadString(hProgInstance, IDS_ALLPICTUREFILES);
+
+        // Get the import filter
+        CSimpleArray<GUID> aguidFileTypesI;
+        CImage::GetImporterFilterString(strFilter, aguidFileTypesI, strAllPictureFiles,
+                                        CImage::excludeDefaultLoad, _T('\0'));
+
+        // Initializing the OPENFILENAME structure for GetOpenFileName
+        ZeroMemory(&ofn, sizeof(ofn));
```

Comment:
remove extra spaces at the left of equals signs.

Reply:
Removed.